### PR TITLE
chore: prepend @aws-amplify to amplify-category-analytics

### PR DESCRIPTION
**Pull request by evcodes**
#### Description of changes
prepending @aws-amplify scope across packages to prevent others from impersonating the Amplify team

#### Description of how you validated changes
running e2e test suite and unit test suite

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-category-analytics/package.json [hunk 1]
```diff
@@ -14,6 +14,9 @@
     "amplify",
     "aws"
   ],
+  "publishConfig": {
```
Comment (lazpavel) [inline packages/amplify-category-analytics/package.json:17]: Do we want it public?

**Review by lazpavel — CHANGES_REQUESTED** [review]
This PR looks that it needs to be rebased

File: packages/amplify-category-analytics/package.json (hunk 1 above)
Comment (evcodes) [reply packages/amplify-category-analytics/package.json:17]: It's already public here: https://www.npmjs.com/package/amplify-category-function. 
If we don't do this then when release it, the change wouldn't be reflected on npm

Comment (sobolk) [reply packages/amplify-category-analytics/package.json:17]: yes, every package that is scoped must be declared public.
fwiw, we had a release fiasco in the past when new package got introduced without that config, see https://github.com/aws-amplify/amplify-cli/pull/11029 .

**Review by sobolk — APPROVED** [review]

**Review by lazpavel — APPROVED** [review]

**Review by sobolk — APPROVED** [review]

**Pull request merged**